ship_traffic/anaconda-project.yml

```diff
@@ -10,27 +10,23 @@ labels:
 user_fields: [labels, skip, maintainers, user_fields]
 
 channels:
-- pyviz
-- conda-forge
+- pyviz/label/dev
```

Comment:
What's label/dev important for? I don't see any dev releases below.

Reply:
Good catch. I can try the solve with just `pyviz` which should work..then I can update the lock again. The original project yaml before I unpinned used a number of alphas.